### ISSUE TITLE
Check the Chinese Error message for mistakes on the Error page

### ISSUE THREAD

**Issue by BennyDeBock** [vapor/docs-cn#21]
It'd be best if someone who actually speaks/writes Chinese goes over the 404 page in the `theme` folder to make sure the text listed is written correctly. (As I have used DeepL to translate to Chinese)

```
404 - 未找到网页

你正在寻找的页面可能已经被删除，它的名字已经改变，或者它暂时无法使用。请稍后再试。
[回到主页](https://cn.docs.vapor.codes/4.0/)
```

This is the text you'd be checking as this text wasn't generated.

**Comment by 0xTim** [comment]
@Jinxiansen are you able to check this?

**Comment by Jinxiansen** [comment]
okay, I will check it at dawn.

**Comment by 0xTim** [comment]
Closed via #16
